client/src/components/Trips/TripView.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react"
+import { Link } from "react-router-dom"
+// import DeleteModal from "../Modal"
+import axios from "axios"
+
+class TripView extends Component {
```

Comment:
I suggest building `TripView` in your next pr. Save your code somewhere, and empty the component.